lib/invoker.js

```diff
@@ -28,14 +29,19 @@ module.exports = (userFunctionUri, port) => {
     const userFunction = lookUpFunction(userFunctionUri);
 
     const server = new grpc.Server();
-    server.addService(services.RiffService, {invoke: invoke(userFunction)});
+    server.addService(services.RiffService, {
+        invoke: invoke(userFunction, () => {
+            logger('Force shutdown');
+            server.forceShutdown();
```

Comment:
I made it more drastic because it is easier to reason about.

A more complex approach would be to keep track of all streams error status and kill the process if and only if all the streams are KO.

Reply:
that's ok for debugging, but it's bad for a released invoker. It would be easy to create a denial of service attack by crafting a message that will error and kill off all other in-progress work.

Reply:
Agreed, an error should not crash the whole process. As a matter of fact, we should test that in the TCK

Reply:
But then, what should be done? 
Let's say a function has two input streams and zips them together to produce an output, what is supposed to happen if one of the input streams errors?

Reply:
Error the **invocation** (not the process). Doesn't that make sense in node-land?

Reply:
these "two" inputs streams are logically defined within the invoker, since they are demuxed from a single grpc invocation. The invocation needs to error without also taking down every other active invocation.